x-pack/plugins/security/server/saved_objects/secure_saved_objects_client_wrapper.ts

```diff
@@ -94,56 +134,146 @@ export class SecureSavedObjectsClientWrapper implements SavedObjectsClientContra
     objects: Array<SavedObjectsBulkCreateObject<T>>,
     options: SavedObjectsBaseOptions = {}
   ) {
-    await this.ensureAuthorized(
-      this.getUniqueObjectTypes(objects),
-      'bulk_create',
-      options.namespace,
-      { objects, options }
-    );
+    const auditor = await this.getScopedAuditor();
+
+    let response: SavedObjectsBulkResponse<T>;
+    try {
+      await this.ensureAuthorized(
+        this.getUniqueObjectTypes(objects),
+        'bulk_create',
+        options.namespace,
+        { objects, options }
+      );
+
+      response = await this.baseClient.bulkCreate(objects, options);
+    } catch (error) {
+      auditor.add(savedObjectCreateEvent, { action: 'saved_object_bulk_create', objects, error });
+      throw error;
+    }
+
+    auditor.add(savedObjectCreateEvent, {
+      action: 'saved_object_bulk_create',
+      objects: response.saved_objects,
+    });
```

Comment:
This is not an issue anymore since we decided to log each object in a bulk operation individually.
To start with we don't log the outcome for write operations but that shouldn't be a problem if we decide to offer that.